sources/packages/api/src/auth/guards/groups.guard.ts

```diff
@@ -33,8 +33,9 @@ export class GroupsGuard implements CanActivate {
     }
     const userToken = user as IUserToken;
     // Check if the user has any of the groups required to have access to the resource.
+    // UserGroups 'aest/user/x' and 'aest/user' are valid, we are here looking for matches, not the exact string.
     return requiredGroups.some((group: string) =>
-      userToken.groups?.includes(group),
+      userToken.groups?.some((userGroup) => userGroup.startsWith(group)),
```

Comment:
The requiredGroups is a list and the userGroup is also a list, using some will return true if one of the required userGroup is available in the group list. It will not check for all the required userGroups, Is the scenario right?

Reply:
yes, `.some()` will check at least one matches the logic. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
👍